aptos-move/block-executor/src/executor.rs

```diff
@@ -390,6 +435,14 @@ where
             if must_skip {
```

Comment:
could merge this check with must_skip || accumulated_gas >= PER_BLOCK_GAS_LIMIT?

Reply:
Actually need to separate them to distinguish reconfiguration from exceeding the per-block gas limit.